source/beamline/AMDetectorTriggerSource.cpp

```diff
@@ -47,11 +47,13 @@ AMDetectorDwellTimeSource::AMDetectorDwellTimeSource(const QString &name, QObjec
 	name_ = name;
 }
 
-void AMDetectorDwellTimeSource::requestSetDwellTime(double dwellSeconds){
+#include <QDebug>
+void AMDetectorDwellTimeSource::requestSetDwellTime(double dwellSeconds)
+{
+	qDebug() << "We are requesting the following dwell time: " << dwellSeconds;
     emit setDwellTime(dwellSeconds);
 }
 
-#include <QDebug>
 void AMDetectorDwellTimeSource:: requestSetDarkCurrentCorrectionTime(double timeSeconds) {
     qDebug() << "Want to emit darkCurrentCorrectionTime as " << timeSeconds << " in " << name();
```

Comment:
And nope.

Sent from my iPhone

On Aug 21, 2015, at 9:39 AM, iainworkman <notifications@github.com<mailto:notifications@github.com>> wrote:


In source/beamline/AMDetectorTriggerSource.cpp<https://github.com/acquaman/acquaman/pull/1541#discussion_r37646769>:

>      emit setDwellTime(dwellSeconds);
>  }
>
> -#include <QDebug>
>  void AMDetectorDwellTimeSource:: requestSetDarkCurrentCorrectionTime(double timeSeconds) {
>      qDebug() << "Want to emit darkCurrentCorrectionTime as " << timeSeconds << " in " << name();


Same re: debug

-
Reply to this email directly or view it on GitHub<https://github.com/acquaman/acquaman/pull/1541/files#r37646769>.